lib/presentation/usage.dart

```diff
@@ -1,140 +1,97 @@
-import 'package:donation_tracker/constants.dart';
-import 'package:donation_tracker/models/usage.dart';
-import 'package:donation_tracker/utils.dart';
 import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:get_it_mixin/get_it_mixin.dart';
 
+import '../constants.dart';
+import '../models/usage.dart';
+import '../utils.dart';
+import 'package:layout/layout.dart';
+
+import 'usage/widgets/user_picture_for_has_date.dart';
+
 class DonationUsages extends StatelessWidget with GetItMixin {
   final ValueNotifier<List<Usage>> usageUpdates;
   final bool hasUsageDates;
 
   DonationUsages(
       {Key? key, required this.usageUpdates, required this.hasUsageDates})
       : super(key: key);
+
   @override
   Widget build(BuildContext context) {
     final usages =
         watch<ValueListenable<List<Usage>>, List<Usage>>(target: usageUpdates);
-
-    return Padding(
-      padding: const EdgeInsets.symmetric(horizontal: 48, vertical: 24),
-      child: Column(crossAxisAlignment: CrossAxisAlignment.stretch, children: [
-        DefaultTextStyle.merge(
-          style: tableHeaderStyle,
-          child: Row(
-            children: [
-              if (hasUsageDates) Expanded(child: Text('Date')),
-              Expanded(child: Text('Amount')),
-              Expanded(child: Text('Usage')),
-              Spacer(),
-              Expanded(child: Text('Receiver')),
-              Spacer(),
+    final columnSpacing = LayoutValue.fromBreakpoint(
+      xs: 4.0,
+      md: 16.0,
+      lg: 32.0,
+      xl: 56.0,
+    );
+    final dataRowHeight = LayoutValue.fromBreakpoint(
+      xs: 80.0,
+      md: 64.0,
+      lg: 56.0,
+      xl: 48.0,
+    );
+    return SingleChildScrollView(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          DataTable(
```

Comment:
As already stated, we should stay with a ListView here.
What you think if we switch for mobiles from a table layout to a card like layout that allows more than one line per entry?
I added a second optional picture to the Usage class in the meantime which we also have to place somewhere if provided.

Reply:
Nice, it would enable the full-text show completely on the screen which is a better alternative